resources/assets/css/_tables.css

```diff
@@ -663,3 +564,77 @@
     > div.table-cell-bg::before {
     @apply bg-transparent;
 }
+
+/* Table with sticky headers you will need to manually override the `top` property for `.table-container table.sticky-headers th` according to your needs */
```

Comment:
@alfonsobries could you split this into 2 lines, please. It's good to try and keep lines shorter (than this) so you don't get the horizontal scroll on GH